akka-typed/src/main/scala/akka/typed/ActorContext.scala

```diff
@@ -62,12 +62,12 @@ trait ActorContext[T] {
    * Create a child Actor from the given [[Props]] under a randomly chosen name.
    * It is good practice to name Actors wherever practical.
    */
-  def spawnAnonymous[U](props: Props[U]): ActorRef[U]
+  def spawnAnonymous[U](behavior: Behavior[U], dispatcher: DispatcherSelector = DispatcherDefault, mailboxCapacity: Int = 1000000): ActorRef[U]
```

Comment:
A potential problem I see with this is that it will be hard to evolve the API, in case we want to add another attribute.